rpc/backend/call_tx.go

```diff
@@ -188,7 +167,7 @@ func (b *Backend) SetTxDefaults(args evmtypes.TransactionArgs) (evmtypes.Transac
 		return args, errors.New("both gasPrice and (maxFeePerGas or maxPriorityFeePerGas) specified")
 	}
 
-	head := b.CurrentHeader()
+	head, _ := b.CurrentHeader() // #nosec G703 -- no need to check error cause we're already checking that head == nil
```

Comment:
What's the issue with still checking the error?

Reply:
this applies to all comments related to `rpc`

since we are forking this part and adding synthetic txs, i just forked it again and applied synthetic txs changes, so all these are coming from evm code

i will need to check rpc folder again after 0.0.3 and can add errors where needed